obspy/signal/spectral_estimation.py

```diff
@@ -1824,6 +1832,31 @@ def plot(self, filename=None, show_coverage=True, show_histogram=True,
                 else:
                     xdata = periods
                 ax.plot(xdata, noise_model, '0.4', linewidth=2, zorder=10)
+        
+        if show_earthquakes:
+            earthquakes=get_earthquakes()
+            for eqk in earthquakes:
```

Comment:
I think across the code base we use `event` for earthquakes most of the time, would be good to adapt this. Maybe also rename the kwarg to `show_events=False` or `show_event_models=False`.

Also, it'd be much more pythonic and way more readable, something like this:

```
for (magnitude, distance), (periods, power) in EARTHQUAKE_MODELS.items():
    ...
```

Reply:
The for loop is great, thanks a lot. But ObsPy uses events because it might be handling any sort of seismic event. Here these are really and only earthquakes carefully selected.